raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -802,8 +802,6 @@ contract TokenNetwork is Utils {
         // for the external APIs
         require(channels[channel_identifier].state == ChannelState.NonExistent);
 
-        require(locks.length > 0);
```

Comment:
IMO this edge case should be unit tested.

Reply:
OK.  No problem adding a test.